README.md

```diff
@@ -111,9 +127,11 @@ sudo yum-config-manager --add-repo https://copr.fedoraproject.org/coprs/petersen
 sudo yum install ghc cabal-install
```

Comment:
It still eats up the last ~15 chars. I'll have to reduce the margins to make it fit.